modules/theme-tools/compat/twentytwentyone.css

```
@@ -7,3 +7,68 @@
 	margin-left: auto;
 	margin-right: auto;
 }
+
+/**
+ * Infinite Scroll
+ */
+
+/* Globally hidden elements when Infinite Scroll is supported and in use. */
+.infinite-scroll .pagination,
+.infinite-scroll.neverending .widget-area,
+.infinite-scroll.neverending footer#colophon {
+	/* Theme Footer (when set to scrolling) */
+	display: none;
+}
+
+/* When Infinite Scroll has reached its end we need to re-display elements that were hidden (via .neverending) before. */
+.infinity-end.neverending .widget-area,
+.infinity-end.neverending footer#colophon {
+	display: block;
+}
+
+.site-main > .infinite-wrap > article > .entry-footer {
+    margin-top: var(--global--spacing-vertical);
+    padding-top: var(--global--spacing-unit);
+    padding-bottom: calc(3 * var(--global--spacing-vertical));
+    border-bottom: var(--separator--height) solid var(--separator--border-color);
+}
+
+.site-main > .infinite-wrap > * {
+    margin-top: calc(3 * var(--global--spacing-vertical));
+    margin-bottom: calc(3 * var(--global--spacing-vertical));
+}
+
+#infinite-handle {
+    text-align: center;
+}
```

You'll probably want to avoid the use of spaces in that file, or someone else editing the file after you may change all spaces into tabs at once when saving.

```suggestion
.site-main > .infinite-wrap > article > .entry-footer {
	margin-top: var(--global--spacing-vertical);
	padding-top: var(--global--spacing-unit);
	padding-bottom: calc(3 * var(--global--spacing-vertical));
	border-bottom: var(--separator--height) solid var(--separator--border-color);
}

.site-main > .infinite-wrap > * {
	margin-top: calc(3 * var(--global--spacing-vertical));
	margin-bottom: calc(3 * var(--global--spacing-vertical));
}

#infinite-handle {
	text-align: center;
}
```

---

Thanks!